### PR TITLE
Web add

### PR DESCRIPTION
**Pull request by arahayrabedian**
solved #22 by adding a submission form on the webs, goes in to db as under moderation mode. 

Reuses existing 'username' field, but without a slack team - I worry this could cause issues in the future, but we can separate out if need by via script.

blank team id should be ok, it's not constrained to not-null, so yeah, we'll see how that goes.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -85,31 +91,33 @@ def process_slack_command(db):
...
-                    "Submit a new excuse to a moderator with "
-                    "`/xqzes add <your text here>`\n"
-                    "e.g: `/xqzes add I was shopping!`"
-        }
-    elif slack_data.text.startswith("add"):
-        # here we want to add a new non-approved excuse
-        excuse_text = slack_data.text.lstrip(" add ")
-        if len(excuse_text) > 140:
+    # small chance text is empty (if a stupid dev is curling manually)
+    if 'text' in slack_data.attributes.keys():
```
Comment (wmgaca-zz) [inline server.py]: `if 'text' in slack_data.attributes:` should be enough

File: server.py [hunk 2]
```diff
@@ -154,6 +162,54 @@ def slack_instructions(db):
     )
 
 
+@route('/submit/')
+@post('/submit/')
+def submit(db):
+
+    # TODO: break this out along with others in to an excuses package.
+    class SubmissionForm(Form):
```
Comment (wmgaca-zz) [inline server.py:170]: A nested class? Not mad, just disappointed...

**Review by wmgaca-zz — APPROVED** [review]
Just two tiny remarks, looks good!

File: server.py (hunk 2 above)
Comment (arahayrabedian) [reply server.py:170]: "TODO" lmao

File: server.py (hunk 1 above)
Comment (arahayrabedian) [reply server.py]: indeed it turns out it is, i had to fiddle to get this to work, so eventually went with what worked. fixed.

Comment (wmgaca-zz) [reply server.py]: Type with me: 

```
/TODO
<enter>
<esc> 
<shift>+<V>
29j
d
:58
<esc>
p
<shift>+<V>
29j
<
:wq
```

File: server.py (hunk 2 above)
Comment (wmgaca-zz) [reply server.py:170]: Type with me :D

```
$ vi server.py

/TODO
<enter>
<esc> 
<shift>+<V>
29j
d
:58
<esc>
p
<shift>+<V>
29j
<
:wq
```

Comment (arahayrabedian) [reply server.py:170]: silly vimster

**Pull request merged**